doc/changelogs/CHANGELOG_V15.md

```diff
@@ -41,6 +42,109 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="15.7.0"></a>
+## 2021-01-22, Version 15.7.0 (Current), @ruyadorno
+
+### Notable changes
+
+* **buffer**:
+  * introduce Blob (James M Snell) [#36811](https://github.com/nodejs/node/pull/36811)
+  * add base64url encoding option (Filip Skokan) [#36952](https://github.com/nodejs/node/pull/36952)
+* **crypto**:
+  * experimental (Ed/X)25519/(Ed/X)448 support (James M Snell) [#36879](https://github.com/nodejs/node/pull/36879)
+* **doc**:
+  * add @iansu to collaborators (Ian Sutherland) [#36951](https://github.com/nodejs/node/pull/36951)
+  * add @RaisinTen to collaborators (Darshan Sen) [#36998](https://github.com/nodejs/node/pull/36998)
```

Comment:
https://github.com/nodejs/node/pull/36934 is missing here (also labelled `notable-change`).

Reply:
thanks @richardlau ❤️ good catch!